torch/_dynamo/optimizations/backends.py

```diff
@@ -752,8 +752,8 @@ def torchxla_trace_once(subgraph):
     def fwd(*args):
         nonlocal subgraph
         nonlocal compiled_graph
-        model = subgraph.model
         if compiled_graph is None:
+            model = subgraph.model
```

Comment:
Try:

1) Moving the model = subgraph.model outside the fwd

or

2) Removing the del

Reply:
ah OK, I can do 1 I guess. What's the motivation of `del` in first place?

Reply:
1 works, updated the pr.